source/UIAHandler/browseMode.py

```diff
@@ -170,6 +186,19 @@ def UIAHeadingQuicknavIterator(
 			searchArea.start = entireDocument.start
 		else:
 			searchArea.end = entireDocument.end
+	if winVersion.getWinVer() >= winVersion.WIN11:

```

Comment:
This winVersion check has to be replicated for every case where UIA Remote is to be used. Rather than having an explicit winVersion check, could you consider having a function or variable on UIAHandler.remote, e.g. `UIAHandler.remote.isSupported`? This also makes it easier to test the pre remote code by changing that variable to False temporarily.